edgelet/hsm-sys/azure-iot-hsm-c/tests/certificate_info_ut/certificate_info_ut.c

```diff
@@ -805,34 +820,22 @@ BEGIN_TEST_SUITE(certificate_info_ut)
 
     TEST_FUNCTION(certificate_info_get_certificate_leaf_win_eol_success)

```

Comment:
This test function is empty now??

Reply:
oh no, editing mistake. glad you are watching...